views/js/scoring/processor/expressions/operators/roundTo.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technlogies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+/**
+ * The roundTo operator processor.
+ * @see http://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element106470
+ *
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([
+    'lodash',
+    'taoQtiItem/scoring/processor/expressions/preprocessor',
+    'taoQtiItem/scoring/processor/errorHandler'
+], function(_, preProcessor, errorHandler){
+    'use strict';
+
+    /**
+     * Process operands and returns roundTo result.
+     * @type {OperatorProcessor}
+     * @exports taoQtiItem/scoring/processor/expressions/operators/roundTo
+     */
+    var roundToProcessor = {
+
+        engines: {
+            significantFigures: function (value, exp) {
+                return decimalAdjust('round', value, exp);
+            },
+            decimalPlaces: function (value, exp) {
+                return decimalAdjust('floor', value, exp);
+            }
+        },
+
+        activeRoundingEngine: null,
+        figures: null,
+
+        constraints : {
+            minOperand  : 1,
+            maxOperand  : 1,
+            cardinality : ['single'],
+            baseType    : ['integer', 'float']
+        },
+
+        operands   : [],
+
+        /**
+         * @returns {?ProcessingValue} a single boolean
+         */
+        process : function(){
+
+            var roundingMode  = this.activeRoundingEngine;
+            var figures = this.figures;
+
```

Comment:
You can get attribute values from `this.expression` ie. `this.expression.figures`  also the attribute is `roundingMode` and not `activeRoundingEngine`